pkgs/development/perl-modules/ImageExifTool/default.nix

```diff
@@ -11,15 +11,19 @@
 
 buildPerlPackage rec {
   pname = "Image-ExifTool";
-  version = "12.84";
+  version = "13.00";
 
   src = fetchurl {
     url = "https://exiftool.org/Image-ExifTool-${version}.tar.gz";
-    hash = "sha256-sfSnx5bS7vI0KIhBOpB5VYzP6g8oi0rR7mUTxxNWEA0=";
+    hash = "sha256-SJV4jzT4NHZfhr5KWtWjJDP1ctdXFg7Ne2Eur17TfoQ=";
   };
 
   nativeBuildInputs = lib.optional stdenv.hostPlatform.isDarwin shortenPerlShebang;
 
+  postPatch = ''
+    patchShebangs exiftool
```

Comment:
Note for reviewers: exiftool shebang has been changed to `/usr/bin/env perl`

https://github.com/exiftool/exiftool/compare/12.84..13.00#diff-472839bdc5e9ec2fb7260a29e39a5fe1fbc9991b793860f184bc4a52b57d0803R1